### ISSUE TITLE
Add MNE v0.21 NIRS features

### ISSUE THREAD

**Issue by rob-luke** [cbrnr/mnelab#149]
Once NIRS support is merged in via #145 I will open a new PR that includes code requiring MNE v0.21.

**Comment by rob-luke** [comment]
First thing will be to add back in `tddr` support as was implemented with the snippets below

```python

        nirs_menu.addSeparator()
        self.actions["apply_tddr"] = nirs_menu.addAction(
                "Apply &TDDR", self.apply_tddr)
        nirs_menu.addSeparator()

            self.actions["apply_tddr"].setEnabled(
                    len(mne.pick_types(self.model.current["data"].info,
                        fnirs="fnirs_od")))


    def apply_tddr(self):
        """Apply TDDR processing to data."""
        self.auto_duplicate()
        self.model.apply_tddr()

    @data_changed
    def apply_tddr(self):
        self.current["data"] = tddr(self.current["data"])
        self.current["name"] += f" (TDDR)"
        self.history.append(
            f"data = mne.preprocessing.nirs."
            f"tddr(data)")
```

**Comment by rob-luke** [comment]
Moved to PR #155
